src/ethereum/ethEngine.js

```diff
@@ -591,6 +625,7 @@ export class EthereumEngine extends CurrencyEngine {
 
   async multicastServers (func: EthFunction, ...params: any): Promise<any> {
     let out = { result: '', server: 'no server' }
+    let funcs, funcs2, url
```

Comment:
These seem to be `const` within each respective case statement. You can use curly braces within case statements to declare local variables and make this clearer:

```js
switch (x) {
  case 1: {
    const local = stuff
    // ...
    break
  }
  case 2: {
    const local = otherStuff
    // ...
    break
  }
}